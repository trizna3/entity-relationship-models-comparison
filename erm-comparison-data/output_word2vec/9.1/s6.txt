marine role-EMPTY
airplane-airplanes
fight model-EMPTY
marine-soldier
rocket type-EMPTY
airplane model-EMPTY
transport model-EMPTY
GENERALIZATION_TO_11_ASSOCIATION:S-fighter planes-airplanes
MERGE_ENTITY_SETS:S-commander-scout
MERGE_ENTITY_SETS:S-commander-medic
