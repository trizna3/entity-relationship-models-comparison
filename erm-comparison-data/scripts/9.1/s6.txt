entity set airplanes {
	"serial number"
	"name"
	"model"
};

entity set "fighter planes" {
	"maximal rocket capacity"
};

entity set "transport planes" {
	"person capacity"
	"maximal load volume"
	"maximal load weight"
};

entity set rocket {
	"type"
};

entity set soldier {
	"name"
	"actual health"
	"maximal health"
	"speed"
	"strength"
	"agility"
	"inteligence"
};

entity set commander {
	"maximal health modifier"
	"health modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

entity set pilot {
	"maximal health modifier"
	"health modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

entity set scout {
	"maximal health modifier"
	"health modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

entity set medic {
	"maximal health modifier"
	"health modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

entity set footman {
	"maximal health modifier"
	"health modifier"
	"strength modifier"
	"agility modifier"
	"inteligence modifier"
};

association "equipment" between 
{
	"fighter planes" MANY
	rocket MANY
}{
	"amount"
};

association "flight list" between 
{
	airplanes MANY
	soldier MANY
}{};

generalization from "fighter planes" to "airplanes";

generalization from "transport planes" to "airplanes";

generalization from "commander" to "soldier";

generalization from "pilot" to "soldier";

generalization from "scout" to "soldier";

generalization from "medic" to "soldier";

generalization from "footman" to "soldier";
