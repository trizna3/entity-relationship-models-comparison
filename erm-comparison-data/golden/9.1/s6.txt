airplane model-model
fight model-fighter planes
rocket type-rocket
transport model-transport planes
airplane-airplanes
marine-soldier
marine role-EMPTY
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-model-airplanes